controllers/machinehealthcheck_targets_test.go

```diff
@@ -23,7 +23,8 @@ import (
 	"time"
 
 	. "github.com/onsi/gomega"
-	gtypes "github.com/onsi/gomega/types"
+	. "github.com/onsi/gomega/types"
```

Comment:
the go `.` import operator is evil.
we removed that pattern from k/k/test btw.

Reply:
i hear you, and agree 100% that it's not great. the main thinking here is to standardize tests around a known pattern and make readability the top concern

We've had multiple PRs go in the past where tests have been hard to read / understand, and the actual test wasn't testing the behaviors we wanted. This is an effort to improve on that.

That said, we shouldn't need `github.com/onsi/gomega/types` with dot-import

Reply:
reverted